data/mships.txt

```diff
@@ -3634,7 +3634,7 @@ ship "SV-4450 Sovereign"
 	explode "large explosion" 60
 	explode "huge explosion" 60
 	"final explode" "final explosion large"
-	description "Sovereign is a massive carrier and worldship of mysterious origins. It is said that this ship was designed many years ago for an expedition to outside of known space. No one knows the fate of the project and only the ships remains today in few private hands."
+	description "The Sovereign is a massive carrier and worldship of mysterious origins. It is said that this ship was designed many years ago for an expedition to outside of known space. No one knows the fate of the project and only a few of the ships remain today in private hands."
```

Comment:
```suggestion
	description "The Sovereign is a massive carrier and worldship of mysterious origins. It is said that this ship was designed many years ago for an expedition to outside of known space. No one knows the fate of the project and only the ships remain today in few private hands."
```
I think that changes the meaning. It's referring to the fact that these ships are the only thing left from the project and owned by a few individuals. (Which also imply that there's only a few of these ships.)